variables.tf

```diff
@@ -0,0 +1,124 @@
+variable "vpc_CIDR_block" {
```

Comment:
Overall the variable descriptions are unsatisfactory. Many are direct proxies. Please review the description of the variables you are proxying.

Recommended to copy and paste these  if direct proxies and also add additional context around which resources these refer to if required

Reply:
done